Description: Add the required ply python module as a dependency
 python3-ply is required by the python scripts, but not declared as a
 dependency.
Author: Andreas Hasenack <andreas@canonical.com>
Bug: https://bugs.isc.org/Public/Bug/Display.html?id=47065
Bug-Debian: https://bugs.debian.org/888463
Last-Update: 2018-01-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/bin/python/setup.py
+++ b/bin/python/setup.py
@@ -14,4 +14,5 @@
       author='Internet Systems Consortium, Inc',
       author_email='bind9-bugs@isc.org',
       license='ISC',
+      requires=['ply'],
       packages=['isc'])
